.travis.yml

```diff
@@ -0,0 +1,66 @@
+language: php
+
+php:
+  - 5.6
+  - 7
+
+env:
+  global:
+    - MODULE_NAME='invisible_recaptcha'
+#    - MODULE_TEST_GROUP='invisible_recaptcha'
```

Comment:
Why is this commented? It is used for running tests.

Reply:
There are no tests for this module yet. If this would be set, the test runner would throw an error, causing the CI script to fail. 
It can be un-commented when tests are implemented.